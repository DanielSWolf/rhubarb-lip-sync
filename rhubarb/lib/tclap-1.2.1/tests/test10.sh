#!/bin/sh

# failure
../examples/test2 -i 10 -s hello > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test10.out; then
	exit 0
else
	exit 1
fi

