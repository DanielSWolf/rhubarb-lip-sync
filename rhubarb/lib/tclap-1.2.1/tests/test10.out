PARSE ERROR:  
             Required argument missing: unTest

Brief USAGE: 
   ../examples/test2  [-f <float>] -i <int> -s <string> [-A] [-C] [-B] [--]
                      [--version] [-h] <string> <string> ...

For complete USAGE and HELP type: 
   ../examples/test2 --help

